dot/network/config.go

```diff
@@ -161,6 +165,19 @@ func (c *Config) checkState() (err error) {
 // service, if a key does not exist or cannot be loaded, it creates a new key
 // using the random seed (if random seed is not set, creates new random key)
 func (c *Config) buildIdentity() error {
+	if c.NodeKey != "" {
+		pkBytes, err := common.HexToBytes("0x" + c.NodeKey)
```

Comment:
nit `pkBytes` -> `privateKeyBytes` (pk could be public key)

also isn't this the `privateKeySeed` perhaps from the looks of the code below?